src/main/java/com/google/devtools/build/lib/query2/cquery/LabelAndConfigurationOutputFormatterCallback.java

```diff
@@ -24,33 +24,46 @@
 
 /** Default Output callback for cquery. Prints a label and configuration pair per result. */
 public class LabelAndConfigurationOutputFormatterCallback extends CqueryThreadsafeCallback {
-
+  private final boolean showKind;
   public LabelAndConfigurationOutputFormatterCallback(
       ExtendedEventHandler eventHandler,
       CqueryOptions options,
       OutputStream out,
       SkyframeExecutor skyframeExecutor,
-      TargetAccessor<ConfiguredTarget> accessor) {
-    super(eventHandler, options, out, skyframeExecutor, accessor);
+      TargetAccessor<ConfiguredTarget> accessor,
+      ConfiguredTargetAccessor configuredTargetAccessor,
+      boolean showKind) {
+    super(eventHandler, options, out, skyframeExecutor, accessor, configuredTargetAccessor, showKind);
```

Comment:
Ok, so what I'm reading from your last error report is compilation fails because the parameters you're passing to CqueryThreadsafeCallback's constructor are different than what the constructor expects:  https://github.com/bazelbuild/bazel/blob/ebca24c525ec93f34bb65053370376082765d800/src/main/java/com/google/devtools/build/lib/query2/cquery/CqueryThreadsafeCallback.java#L48-L53

The link above doesn't know what to do with an extra `ConfiguredTargetAccessor` argument.

But I just realized something that can make this all simpler! You're already accepting a `TargetAccessor<ConfiguredTarget> accessor` above, and `ConfiguredTargetAccessor` is already an instance of `TargetAccessor`!

https://github.com/bazelbuild/bazel/blob/f2cb3c69413c74569f19232f89a5f0e86d4f15ac/src/main/java/com/google/devtools/build/lib/query2/cquery/ConfiguredTargetAccessor.java#L56

So you don't have to pass the new `ConfiguredTargetAccessor configuredTargetAccessor` parameter at all (my bad). Instead, simply call `accessor.getTargetKind()` in `processOutput` below. And there's no need to change anything  here in the constructor.